csfieldguide/templates/chapters/index.html

```diff
@@ -14,7 +14,7 @@ <h1 id="chapters" class="section-heading anchor-link">Chapters</h1>
 {% block content %}
 
   {% if all_chapters %}
-      
+
```

Comment:
Reason for extra whitespace? This seems to be making page size bigger for no gain.